src/engraving/dom/excerpt.cpp

```diff
@@ -1533,7 +1533,7 @@ void Excerpt::cloneStaff2(Staff* srcStaff, Staff* dstStaff, const Fraction& star
                         continue;
                     }
                     bool systemObject = e->systemFlag() && e->track() == 0;
-                    bool alreadyCloned = bool(e->findLinkedInScore(score));
+                    bool alreadyCloned = bool(e->findLinkedInScore(score)) && e->parent() == ns;
```

Comment:
Should this check be as follows?
```suggestion
                    EngravingItem* linkedElement = e->findLinkedInScore(score);
                    bool alreadyCloned = linkedElement && linkedElement->parent() == ns;
```
If `e` comes from `oseg`'s annotations, then won't this condition always be false?